packages/coinstac-ui/app/render/routes.js

```diff
@@ -27,7 +27,7 @@ export default (
       <IndexRoute component={DashboardHome} />
       <Route path="/consortia" component={DashboardConsortia} />
       <Route path="/consortia/new" component={ConsortiumController} />
-      <Route path="/consortia/:consortiumId" component={ConsortiumController} />
+      <Route path="/consortia/:consortiumId(/:computationId)" component={ConsortiumController} />
```

Comment:
When I wrote this, I was just using the optional route param _computationId_ (now _resultId_) to autoexpand the respective result accordion and not to jump to that section of the page, though the capacity was there. I've now built it to work that way using a timeout, which feels hacky, but I've retained the _resultId_ as a route param. This might not follow the web paradigm for anchoring to a section of a "view", but since that's not exclusively its purpose, is that okay?